### PR TITLE
add tuya logo

### PR DESCRIPTION
**Pull request by erdun**
Add tuya logo in `Istio in Action`

### CODE REVIEW

File: content/en/about/community/customers/index.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ Tons of people are using Istio. Maybe you should too?
     {{< company_logo link="https://www.trulia.com" logo="./trulia.svg" alt="Trulia" >}}
     {{< company_logo link="https://www.ibm.com/watson" logo="./watson.png" alt="IBM Watson" >}}
     {{< company_logo link="https://ygrene.com" logo="./ygrene.jpeg" alt="Ygrene Energy Fund" >}}
+    {{< company_logo link="https://www.tuya.com" logo="./tuya.png" alt="Tuya Inc" >}}
```
Comment (frankbu) [inline content/en/about/community/customers/index.md:35]: These entries are in alphabetical order. Please move this up (after Trulia, before IBM Watson).

**Comment by frankbu** [comment]
Please only make this change in the `en` version of the docs. The other languages will be synchronized by the translation teams after this has merged.

File: content/en/about/community/customers/index.md (hunk 1 above)
Comment (erdun) [reply content/en/about/community/customers/index.md:35]: 👌

**Comment by howardjohn** [comment]
@googlebot

**Pull request merged**